test/analytics/config.ts

```diff
@@ -34,8 +34,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
       ...functionalConfig.get('kbnTestServer'),
       serverArgs: [
         ...functionalConfig.get('kbnTestServer.serverArgs'),
-        // Disabling telemetry, so it doesn't call opt-in before the tests run.
-        '--telemetry.enabled=false',
+        '--telemetry.optIn=true',
```

Comment:
We can't disable telemetry anymore, so we need to optIn and rewrite the tests knowing that `optIn: true` from the beginning of the tests.